src/Microsoft.AspNet.Http/Features/IFeatureManager.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Http.Features
+{
+    public interface IFeatureManager
```

Comment:
No, but would be implementing without contract; which is probably fine :)
Removed.